strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/npm/NpmArtifactControllerTest.java

```diff
@@ -125,4 +125,53 @@ public void testPackageCommonFlow(@NpmRepository(repositoryId = REPOSITORY_RELEA
                .header(HttpHeaders.CONTENT_LENGTH, equalTo(String.valueOf(Files.size(packagePath))));
     }
 
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class,
+            ArtifactManagementTestExecutionListener.class })
```

Comment:
Cosmetic: Indentation:
```suggestion
    @ExtendWith({ RepositoryManagementTestExecutionListener.class,
                  ArtifactManagementTestExecutionListener.class })
```